satpy/tests/test_writers.py

```diff
@@ -260,7 +258,6 @@ class TestEnhancerUserConfigs(_BaseCustomEnhancementConfigTests):
 
         """,
         ENH_ENH_FN: """
-sensor_name: visir/test_sensor
```

Comment:
Why remove the sensor info here? We have them in the yaml, right?

Reply:
We don't have them in enhancement files. I don't know why I added it here in the first place. The composites use this but it has no effect here for enhancements.